cpp/ql/test/library-tests/ir/range-analysis/SimpleRangeAnalysis_tests.cpp

```diff
@@ -39,37 +39,37 @@ int test4() {
   int i = 0;
   int total = 0;
   for (i = 0; i < 2; i = i+1) {
-    range(i); // $ range=<=1 range=>=0
+    range(i); // $ range=<=1 MISSING: range=>=0
```

Comment:
Are we losing these results because we're now claiming that `i+1` on line 41 overflows?

In general: Should we add an InlineExpectationsTest that highlights all the places where we're claiming an overflow might occur. This would give me a lot more confidence in what's actually going on here.